### PR TITLE
Images uploaded fix

### PR DESCRIPTION
**Pull request by colmak**

### CODE REVIEW

**Comment by colmak** [comment]
needs fixing

**Comment by colmak** [comment]
nvm it was my database I had to migrate as I had to reset it

File: smart_emailApp/views.py [hunk 1]
```diff
@@ -173,7 +179,7 @@ def edit_task(request, id):
             intance.recipients = members_list
 
         intance.save()
-        return redirect('view_tasks')
+        return redirect('master_home')
```
Comment (giosroom99) [inline smart_emailApp/views.py:182]: **We want to redirect to view_tasks  not back home**

File: smart_emailApp/views.py [hunk 2]
```diff
@@ -134,7 +140,7 @@ def create_task(request):
 
             instance.save()
 
-            return redirect('view_tasks')
+            return redirect('master_home')
```
Comment (giosroom99) [inline smart_emailApp/views.py:143]: We want to redirect to view_tasks  not back home

File: smart_emailApp/views.py [hunk 3]
```diff
@@ -90,15 +89,20 @@ def scheduled_email(request):
...
 
 def create_email(request):
-    form = CreateEmailForm(request.POST, request.FILES)
     if request.method == 'POST':
+        form = CreateEmailForm(request.POST, request.FILES)
         if form.is_valid():
-            # handle_uploaded_file(request.FILES["Stage_image"])
             instance = form.save(commit=False)
+            # Save the product images
+            image = form.cleaned_data.get('image')
```
Comment (giosroom99) [inline smart_emailApp/views.py:97]: I didnt know that what I had to do, I never had to this, but if it works that fine.

**Review by giosroom99 — CHANGES_REQUESTED** [review]
See comment,
After the user creates or edit  tasks and email we should not bring them back to home screen, we should take them to emails views or tasks views

**Pull request merged**